src/main/drivers/accgyro/accgyro_spi_icm426xx.c

```diff
@@ -168,10 +176,61 @@ static aafConfig_t aafLUT42605[AAF_CONFIG_COUNT] = {  // see table in section 5.
     [AAF_CONFIG_1962HZ] = { 63, 3968,  3 }, // 995 Hz is the max cutoff on the 42605
 };
 
+#if defined(USE_GYRO_CLKIN)
+static pwmOutputPort_t pwmGyroClk = {0};
+
+static bool initExternalClock(void)
+{
+    const ioTag_t tag = IO_TAG(GYRO_CLKIN_PIN);
```

Comment:
```suggestion
    if (gyro.gyroSensor1.gyroDev.dev != dev) {
        // only gyro1 clkin supported.
        return false;
    }
    const ioTag_t tag = gyroDeviceConfig(0)->clkIn;
```

This is terrible hack, but fixing dual gyro will affect a lot of code. It can be done in follow-up PR, after proving that clkin is good idea.